Changes

```diff
@@ -161,6 +161,9 @@ Working version
 
 ### Bug fixes:
 
+- #7156: Create Mechanism so that OCaml Top Level uses custom printers 
```

Comment:
lowercase is more standard, and "exception" should be in the description: "make the toplevel use custom exception printers if they are available>